src/main/java/org/reaktivity/nukleus/http2/internal/Http2Stream.java

```diff
@@ -190,10 +189,7 @@ void onAbort(long traceId)
     void onReset(long networkReplyTraceId)
     {
         // reset the response stream
-        if (applicationReplyThrottle != null)
-        {
-            factory.doReset(applicationReplyThrottle, applicationRouteId, applicationReplyId, networkReplyTraceId);
-        }
+        factory.doReset(applicationInitial, applicationRouteId, applicationReplyId, networkReplyTraceId);
```

Comment:
This should also cleanup the correlations map entry to avoid a memory leak, now that the `BEGIN` response will not arrive to consume the correlations map entry.

Let's also remote `applicationReplyThrottle` and use `applicationInitial` instead.